packages/gatsby-theme-intranet/src/components/DnDMenu/DnDMenuDisplayView.js

```diff
@@ -33,7 +35,7 @@ export default function DnDMenuDisplayView({ ...restProps }) {
           aria-pressed={displayAllItems}
           onClick={() => setDisplayAllItems((value) => !value)}
         >
-          {!displayAllItems ? "Show more" : "Show less"}
+          {!displayAllItems ? t("showMoreToolsLabel") : t("showLessToolsLabel")}
```

Comment:
I think these should be provided via props instead, since the component is used for both bookmarks and tools